src/Test.CoreLib/src/System/ReadOnlySpan.cs

```diff
@@ -0,0 +1,117 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using Internal.Runtime.CompilerHelpers;
```

Comment:
The `ThowHelpers` in this namespace are for compiler use (that's why they're all private). We don't call them from BCL code.